src/data/real/ennreal.lean

```diff
@@ -1489,6 +1505,12 @@ lemma to_real_prod {ι : Type*} {s : finset ι} {f : ι → ℝ≥0∞} :
   (∏ i in s, f i).to_real = ∏ i in s, (f i).to_real :=
 to_real_hom.map_prod _ _
 
+lemma to_real_inv (a : ℝ≥0∞) : (a⁻¹).to_real = (a.to_real)⁻¹ :=
+by { simp_rw ennreal.to_real, norm_cast, exact to_nnreal_inv a, }
```

Comment:
Can you prove this with `monoid_with_zero_hom.map_inv'`? I'm pretty sure `to_real_hom` can be promoted to that stronger type.

Reply:
ennreal doesn't have the required instance: `failed to synthesize type class instance for group_with_zero ℝ≥0∞`

Reply:
I mean that I can indeed promote `to_nnreal_hom` to a `monoid_with_zero_hom`, but I cannot use `map_inv'`